R/tokens.R

```diff
@@ -341,17 +351,20 @@ tokens.tokens <-  function(x,
         x <- tokens_split(x, "\\p{Pd}", valuetype = "regex", remove_separator = FALSE)
     }
     
+    if (remove_separators)
+        x <- tokens_remove(x, "^[\\p{Z}\\p{C}]$", valuetype = "regex")
+    
     # removals
     removals <- compile_removals_regex(remove_separators = remove_separators,
                                        remove_punct = remove_punct,
                                        remove_symbols = remove_symbols,
                                        remove_numbers = remove_numbers,
                                        remove_url = remove_url)
-    if (length(removals$regex_to_remove)) {
+    if (length(removals$regex_to_remove) || remove_separators) {
```

Comment:
A small hack to print the separators removal message, even though it's been removed at line 358.